core/src/main/java/com/scalar/db/storage/multistorage/MultiStorageAdmin.java

```diff
@@ -182,6 +184,16 @@ public void addNewColumnToTable(
     getAdmin(namespace, table).addNewColumnToTable(namespace, table, columnName, columnType);
   }
 
+  @Override
+  public Set<String> getNamespaceNames() throws ExecutionException {
+    Set<String> namespaceNames = new HashSet<>();
+    for (DistributedStorageAdmin admin : admins) {
+      namespaceNames.addAll(admin.getNamespaceNames());
+    }
+
+    return namespaceNames;
+  }
```

Comment:
It looks like this method returns namespaces that are not in the namespace mapping. I'm not sure but is that okay?

Reply:
I see. Thank you for pointing this out. 
The configuration `multi-storage.table_mapping`  was to address the case where a given namespace would contain Scalar DB table and other non-Scalar DB tables? 
In that case, returning namespaces only from the `multi_storage.namespace_mapping`config is probably more appropriate.  
Concerning the `multi_storage.default_storage` admin, do you think we should return its namespaces ?

Reply:
`multi-storage.table_mapping` is a deprecated config, so let's ignore it here.

Maybe returning namespaces from the `multi_storage.namespace_mappingconfig` config and the `multi_storage.default_storage` admin is more appropriate?

I think, for example, if users get namespaces from `getNamespaceNames()` but they face a `namespace not found` error when they call `getNamespaceTableNames()` for the namespace, it's very confusing behavior. This happens when `getNamespaceNames()` returns namespaces that are neither in the mapping nor the `multi_storage.default_storage` admin.

From this point of view, I feel that returning namespaces from the `multi_storage.namespace_mappingconfig` config and the `multi_storage.default_storage` admin is more appropriate.

What do you think?

Reply:
> From this point of view, I feel that returning namespaces from the multi_storage.namespace_mappingconfig config and the multi_storage.default_storage admin is more appropriate.

I agree. I will revise it accordingly.